tests/driver_blemin/main.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the NRF51822 BLE minimal radio driver (blemin)
+ *
+ * @author      Jan Wagner <mail@jwagner.eu>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "shell.h"
+#include "posix_io.h"
+#include "board_uart0.h"
+#include "net/ble_ll.h"
+#include "net/ng_netbase.h"
+#include "net/ng_pktdump.h"
+
+#define SHELL_BUFSIZE       (UART0_BUFSIZE)
+
+//static char dump_stack[KERNEL_CONF_STACKSIZE_MAIN];
+static char nomac_stack[THREAD_STACKSIZE_DEFAULT];
+
+int main(void)
+{
+    shell_t shell;
+    ng_netdev_t dev;
+
+    puts("\nManual test for the minimal NRF51822 BLE radio driver\n");
+    puts("Use the 'ifconfig' and 'blesnd' shell commands to verify the driver");
+
+    /* initialize network device */
+    /* interface should have already been initialized by auto-init module */
+    nrf51_ble_init(&dev);
+    ble_ll_init(nomac_stack, sizeof(nomac_stack), 5, "ble_ll", &dev);
```

Comment:
I think it would increas the readability if you would define the priority like e.g. for the [at86rf2xx transceiver](https://github.com/RIOT-OS/RIOT/blob/master/sys/auto_init/netif/auto_init_ng_at86rf2xx.c#L37)